Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/usr/local/lib/python3.8/dist-packages/nbclient/client.py", line 1112, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/usr/local/lib/python3.8/dist-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/usr/local/lib/python3.8/dist-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/usr/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/usr/local/lib/python3.8/dist-packages/nbclient/client.py", line 553, in async_execute
    await self.async_execute_cell(
  File "/usr/local/lib/python3.8/dist-packages/nbclient/client.py", line 857, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/usr/local/lib/python3.8/dist-packages/nbclient/client.py", line 760, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
ds_p5 = cat['P5']['NOSE_BOOM']['ACLOUD_P5_RF07'].read()
ds_p5
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mUnicodeDecodeError[0m                        Traceback (most recent call last)
[0;32m/tmp/ipykernel_1071751/2781733681.py[0m in [0;36m<module>[0;34m[0m
[0;32m----> 1[0;31m [0mds_p5[0m [0;34m=[0m [0mcat[0m[0;34m[[0m[0;34m'P5'[0m[0;34m][0m[0;34m[[0m[0;34m'NOSE_BOOM'[0m[0;34m][0m[0;34m[[0m[0;34m'ACLOUD_P5_RF07'[0m[0;34m][0m[0;34m.[0m[0mread[0m[0;34m([0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      2[0m [0mds_p5[0m[0;34m[0m[0;34m[0m[0m

[0;32m/usr/local/lib/python3.8/dist-packages/intake/source/csv.py[0m in [0;36mread[0;34m(self)[0m
[1;32m    140[0m [0;34m[0m[0m
[1;32m    141[0m     [0;32mdef[0m [0mread[0m[0;34m([0m[0mself[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 142[0;31m         [0mself[0m[0;34m.[0m[0m_get_schema[0m[0;34m([0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    143[0m         [0;32mreturn[0m [0mself[0m[0;34m.[0m[0m_dataframe[0m[0;34m.[0m[0mcompute[0m[0;34m([0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m    144[0m [0;34m[0m[0m

[0;32m/usr/local/lib/python3.8/dist-packages/intake/source/csv.py[0m in [0;36m_get_schema[0;34m(self)[0m
[1;32m    125[0m [0;34m[0m[0m
[1;32m    126[0m         [0;32mif[0m [0mself[0m[0;34m.[0m[0m_dataframe[0m [0;32mis[0m [0;32mNone[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 127[0;31m             [0mself[0m[0;34m.[0m[0m_open_dataset[0m[0;34m([0m[0murlpath[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    128[0m [0;34m[0m[0m
[1;32m    129[0m         [0mdtypes[0m [0;34m=[0m [0mself[0m[0;34m.[0m[0m_dataframe[0m[0;34m.[0m[0m_meta[0m[0;34m.[0m[0mdtypes[0m[0;34m.[0m[0mto_dict[0m[0;34m([0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;32m/usr/local/lib/python3.8/dist-packages/intake/source/csv.py[0m in [0;36m_open_dataset[0;34m(self, urlpath)[0m
[1;32m     98[0m [0;34m[0m[0m
[1;32m     99[0m         [0;32mif[0m [0mself[0m[0;34m.[0m[0mpattern[0m [0;32mis[0m [0;32mNone[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 100[0;31m             self._dataframe = dask.dataframe.read_csv(
[0m[1;32m    101[0m                 [0murlpath[0m[0;34m,[0m [0mstorage_options[0m[0;34m=[0m[0mself[0m[0;34m.[0m[0m_storage_options[0m[0;34m,[0m[0;34m[0m[0;34m[0m[0m
[1;32m    102[0m                 **self._csv_kwargs)

[0;32m/usr/lib/python3/dist-packages/dask/dataframe/io/csv.py[0m in [0;36mread[0;34m(urlpath, blocksize, collection, lineterminator, compression, sample, enforce, assume_missing, storage_options, include_path_column, **kwargs)[0m
[1;32m    564[0m         [0;34m**[0m[0mkwargs[0m[0;34m[0m[0;34m[0m[0m
[1;32m    565[0m     ):
[0;32m--> 566[0;31m         return read_pandas(
[0m[1;32m    567[0m             [0mreader[0m[0;34m,[0m[0;34m[0m[0;34m[0m[0m
[1;32m    568[0m             [0murlpath[0m[0;34m,[0m[0;34m[0m[0;34m[0m[0m

[0;32m/usr/lib/python3/dist-packages/dask/dataframe/io/csv.py[0m in [0;36mread_pandas[0;34m(reader, urlpath, blocksize, collection, lineterminator, compression, sample, enforce, assume_missing, storage_options, include_path_column, **kwargs)[0m
[1;32m    442[0m [0;34m[0m[0m
[1;32m    443[0m     [0;31m# Use sample to infer dtypes and check for presence of include_path_column[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 444[0;31m     [0mhead[0m [0;34m=[0m [0mreader[0m[0;34m([0m[0mBytesIO[0m[0;34m([0m[0mb_sample[0m[0;34m)[0m[0;34m,[0m [0;34m**[0m[0mkwargs[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    445[0m     [0;32mif[0m [0minclude_path_column[0m [0;32mand[0m [0;34m([0m[0minclude_path_column[0m [0;32min[0m [0mhead[0m[0;34m.[0m[0mcolumns[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m    446[0m         raise ValueError(

[0;32m/usr/local/lib/python3.8/dist-packages/pandas/io/parsers.py[0m in [0;36mread_csv[0;34m(filepath_or_buffer, sep, delimiter, header, names, index_col, usecols, squeeze, prefix, mangle_dupe_cols, dtype, engine, converters, true_values, false_values, skipinitialspace, skiprows, skipfooter, nrows, na_values, keep_default_na, na_filter, verbose, skip_blank_lines, parse_dates, infer_datetime_format, keep_date_col, date_parser, dayfirst, cache_dates, iterator, chunksize, compression, thousands, decimal, lineterminator, quotechar, quoting, doublequote, escapechar, comment, encoding, dialect, error_bad_lines, warn_bad_lines, delim_whitespace, low_memory, memory_map, float_precision, storage_options)[0m
[1;32m    608[0m     [0mkwds[0m[0;34m.[0m[0mupdate[0m[0;34m([0m[0mkwds_defaults[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m    609[0m [0;34m[0m[0m
[0;32m--> 610[0;31m     [0;32mreturn[0m [0m_read[0m[0;34m([0m[0mfilepath_or_buffer[0m[0;34m,[0m [0mkwds[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    611[0m [0;34m[0m[0m
[1;32m    612[0m [0;34m[0m[0m

[0;32m/usr/local/lib/python3.8/dist-packages/pandas/io/parsers.py[0m in [0;36m_read[0;34m(filepath_or_buffer, kwds)[0m
[1;32m    460[0m [0;34m[0m[0m
[1;32m    461[0m     [0;31m# Create the parser.[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 462[0;31m     [0mparser[0m [0;34m=[0m [0mTextFileReader[0m[0;34m([0m[0mfilepath_or_buffer[0m[0;34m,[0m [0;34m**[0m[0mkwds[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    463[0m [0;34m[0m[0m
[1;32m    464[0m     [0;32mif[0m [0mchunksize[0m [0;32mor[0m [0miterator[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m

[0;32m/usr/local/lib/python3.8/dist-packages/pandas/io/parsers.py[0m in [0;36m__init__[0;34m(self, f, engine, **kwds)[0m
[1;32m    817[0m             [0mself[0m[0;34m.[0m[0moptions[0m[0;34m[[0m[0;34m"has_index_names"[0m[0;34m][0m [0;34m=[0m [0mkwds[0m[0;34m[[0m[0;34m"has_index_names"[0m[0;34m][0m[0;34m[0m[0;34m[0m[0m
[1;32m    818[0m [0;34m[0m[0m
[0;32m--> 819[0;31m         [0mself[0m[0;34m.[0m[0m_engine[0m [0;34m=[0m [0mself[0m[0;34m.[0m[0m_make_engine[0m[0;34m([0m[0mself[0m[0;34m.[0m[0mengine[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    820[0m [0;34m[0m[0m
[1;32m    821[0m     [0;32mdef[0m [0mclose[0m[0;34m([0m[0mself[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m

[0;32m/usr/local/lib/python3.8/dist-packages/pandas/io/parsers.py[0m in [0;36m_make_engine[0;34m(self, engine)[0m
[1;32m   1048[0m             )
[1;32m   1049[0m         [0;31m# error: Too many arguments for "ParserBase"[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m
[0;32m-> 1050[0;31m         [0;32mreturn[0m [0mmapping[0m[0;34m[[0m[0mengine[0m[0;34m][0m[0;34m([0m[0mself[0m[0;34m.[0m[0mf[0m[0;34m,[0m [0;34m**[0m[0mself[0m[0;34m.[0m[0moptions[0m[0;34m)[0m  [0;31m# type: ignore[call-arg][0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m   1051[0m [0;34m[0m[0m
[1;32m   1052[0m     [0;32mdef[0m [0m_failover_to_python[0m[0;34m([0m[0mself[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m

[0;32m/usr/local/lib/python3.8/dist-packages/pandas/io/parsers.py[0m in [0;36m__init__[0;34m(self, f, **kwds)[0m
[1;32m   2308[0m                 [0mself[0m[0;34m.[0m[0mnum_original_columns[0m[0;34m,[0m[0;34m[0m[0;34m[0m[0m
[1;32m   2309[0m                 [0mself[0m[0;34m.[0m[0munnamed_cols[0m[0;34m,[0m[0;34m[0m[0;34m[0m[0m
[0;32m-> 2310[0;31m             ) = self._infer_columns()
[0m[1;32m   2311[0m         [0;32mexcept[0m [0;34m([0m[0mTypeError[0m[0;34m,[0m [0mValueError[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m   2312[0m             [0mself[0m[0;34m.[0m[0mclose[0m[0;34m([0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;32m/usr/local/lib/python3.8/dist-packages/pandas/io/parsers.py[0m in [0;36m_infer_columns[0;34m(self)[0m
[1;32m   2724[0m         [0;32melse[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m   2725[0m             [0;32mtry[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0;32m-> 2726[0;31m                 [0mline[0m [0;34m=[0m [0mself[0m[0;34m.[0m[0m_buffered_line[0m[0;34m([0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m   2727[0m [0;34m[0m[0m
[1;32m   2728[0m             [0;32mexcept[0m [0mStopIteration[0m [0;32mas[0m [0merr[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m

[0;32m/usr/local/lib/python3.8/dist-packages/pandas/io/parsers.py[0m in [0;36m_buffered_line[0;34m(self)[0m
[1;32m   2809[0m             [0;32mreturn[0m [0mself[0m[0;34m.[0m[0mbuf[0m[0;34m[[0m[0;36m0[0m[0;34m][0m[0;34m[0m[0;34m[0m[0m
[1;32m   2810[0m         [0;32melse[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0;32m-> 2811[0;31m             [0;32mreturn[0m [0mself[0m[0;34m.[0m[0m_next_line[0m[0;34m([0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m   2812[0m [0;34m[0m[0m
[1;32m   2813[0m     [0;32mdef[0m [0m_check_for_bom[0m[0;34m([0m[0mself[0m[0;34m,[0m [0mfirst_row[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m

[0;32m/usr/local/lib/python3.8/dist-packages/pandas/io/parsers.py[0m in [0;36m_next_line[0;34m(self)[0m
[1;32m   2906[0m [0;34m[0m[0m
[1;32m   2907[0m             [0;32mwhile[0m [0;32mTrue[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0;32m-> 2908[0;31m                 [0morig_line[0m [0;34m=[0m [0mself[0m[0;34m.[0m[0m_next_iter_line[0m[0;34m([0m[0mrow_num[0m[0;34m=[0m[0mself[0m[0;34m.[0m[0mpos[0m [0;34m+[0m [0;36m1[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m   2909[0m                 [0mself[0m[0;34m.[0m[0mpos[0m [0;34m+=[0m [0;36m1[0m[0;34m[0m[0;34m[0m[0m
[1;32m   2910[0m [0;34m[0m[0m

[0;32m/usr/local/lib/python3.8/dist-packages/pandas/io/parsers.py[0m in [0;36m_next_iter_line[0;34m(self, row_num)[0m
[1;32m   2966[0m             [0;31m# assert for mypy, data is Iterator[str] or None, would error in next[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m
[1;32m   2967[0m             [0;32massert[0m [0mself[0m[0;34m.[0m[0mdata[0m [0;32mis[0m [0;32mnot[0m [0;32mNone[0m[0;34m[0m[0;34m[0m[0m
[0;32m-> 2968[0;31m             [0;32mreturn[0m [0mnext[0m[0;34m([0m[0mself[0m[0;34m.[0m[0mdata[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m   2969[0m         [0;32mexcept[0m [0mcsv[0m[0;34m.[0m[0mError[0m [0;32mas[0m [0me[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m   2970[0m             [0;32mif[0m [0mself[0m[0;34m.[0m[0mwarn_bad_lines[0m [0;32mor[0m [0mself[0m[0;34m.[0m[0merror_bad_lines[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m

[0;32m/usr/local/lib/python3.8/dist-packages/pandas/io/parsers.py[0m in [0;36m_read[0;34m()[0m
[1;32m   2445[0m [0;34m[0m[0m
[1;32m   2446[0m             [0;32mdef[0m [0m_read[0m[0;34m([0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0;32m-> 2447[0;31m                 [0mline[0m [0;34m=[0m [0mf[0m[0;34m.[0m[0mreadline[0m[0;34m([0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m   2448[0m                 [0mpat[0m [0;34m=[0m [0mre[0m[0;34m.[0m[0mcompile[0m[0;34m([0m[0msep[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m   2449[0m [0;34m[0m[0m

[0;32m/usr/lib/python3.8/codecs.py[0m in [0;36mdecode[0;34m(self, input, final)[0m
[1;32m    320[0m         [0;31m# decode input (taking the buffer into account)[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m
[1;32m    321[0m         [0mdata[0m [0;34m=[0m [0mself[0m[0;34m.[0m[0mbuffer[0m [0;34m+[0m [0minput[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 322[0;31m         [0;34m([0m[0mresult[0m[0;34m,[0m [0mconsumed[0m[0;34m)[0m [0;34m=[0m [0mself[0m[0;34m.[0m[0m_buffer_decode[0m[0;34m([0m[0mdata[0m[0;34m,[0m [0mself[0m[0;34m.[0m[0merrors[0m[0;34m,[0m [0mfinal[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    323[0m         [0;31m# keep undecoded input until the next call[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m
[1;32m    324[0m         [0mself[0m[0;34m.[0m[0mbuffer[0m [0;34m=[0m [0mdata[0m[0;34m[[0m[0mconsumed[0m[0;34m:[0m[0;34m][0m[0;34m[0m[0;34m[0m[0m

[0;31mUnicodeDecodeError[0m: 'utf-8' codec can't decode byte 0x8b in position 1: invalid start byte
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x8b in position 1: invalid start byte

